modules/unsupported/wfs-ng/src/main/java/org/geotools/data/wfs/WFSFeatureReader.java

```diff
@@ -37,7 +38,12 @@
 
     private SimpleFeatureType featureType;
 
-    public WFSFeatureReader(final GetParser<SimpleFeature> parser) throws IOException {
+    private GetFeatureResponse response;
+
+    public WFSFeatureReader(
```

Comment:
This is an API change without a deprecation cycle, I think the old constructor can be deprecated and call this with `null` for the response

Reply:
Okay, I'll change that (though it's a package private class never intended to be called directly, you never know I guess).

Reply:
that skipped my attention, in that case it would be OK I guess